docs/_articles/en/codebuilder/cb-start.md

```diff
@@ -1,44 +1,49 @@
-4. Give your project a name and choose **Standard** for project type.
+2. Click **Launch** to launch your Code Builder environment. A new environment is created for you the first time to launch. We've also created an empty Salesforce DX project to help you get started.
 
-Next, connect an org to your Code Builder environment. You can choose to connect to a Salesforce org or a sandbox org.
+![Code Builder Dashboard](../../../images/cb_dashboard.png)
 
-1. Log in to your development environment, click **Allow** to grant access, then click **Next**.
-2. Choose an alias (nickname) for your org. If you work on many orgs, make sure the nickname helps you identify the org quickly in the future. 
-Your new Code Builder environment is now available on your dashboard.
-3. Click **Launch** to launch Code Builder in a new tab. 
-4. Now sit back and relax for a few minutes while Code Builder creates and configures your workspace. Code Builder might take a fews minutes to start up when it initially sets up.
 
 **Tip**: Once you have created your Code Builder environment, launch it from the dashboard at any time, or bookmark it for fast access.
 
+## Connect to a Salesforce Org
+
+Next, connect an org to your Code Builder environment. During the course of development, you'll use different types of orgs for different stages. For example, it's common to use a Developer sandbox or Development Edition org during the development phase, and move to other sandbox types for integration, testing, and staging. Eventually, you'll deploy your changes to a production org. You can connect Code Builder to any of these orgs to deploy or retrieve metadata.
+
+To connect to an org the first time you launch Code Builder:
+
+1. In your Code Builder environment, click **Connect an Org** to connect to the Salesforce org you want to work in.
+2. Enter the login URL or select the org you want to log into.
+3. Enter an alias for the org, for example, dev_pro_sandbox or my_playground.
+4. A code is displayed in a text box. Click **Connect**.
+5. Log in with the relevant username and password. Click **Allow**.
+6. Click **Continue**. You’re now connected to an org, and its name is visible in the status bar at the bottom in the Code Builder tab.
+
+
+You can change the org you are connected to by clicking on the name of the current org to bring up the Command Palette and run **SFDX: Authorize an Org**. again.
+
+**Note**: After you authorize an org, we take care of future authorizations so you don't have to continually log in. Just click the org’s name and then choose the org from the list.
+
 ## Let's Take A Quick Tour
+
 You’re now in your developer environment in Code Builder. Code Builder has a rich IDE by VS Code, plus it gives you easy access to cool Salesforce development-specific tools through the Salesforce extension pack. Let’s take a quick tour and get to know the lay of the land. Your screen looks something like this:
 
 ![code_builder_ui](../../../images/code_builder_ui.jpg)
```

Comment:
@sbudhirajadoc - Bonus points if we can update the screenshot to be from Code Builder (or photoshop in the resource panel - the icon that is a cloud with a question mark.). I think you inherited this image and we don't have the original so not a blocker.

Reply:
Yes, on it. getting one from Meg. :-)

Reply:
updated from a screenshot.